server/src/test/java/org/elasticsearch/cluster/coordination/ElectionSchedulerFactoryTests.java

```diff
@@ -68,7 +68,8 @@ private void assertElectionSchedule(
                     "retrying master election after [0] failed attempts"
                 )
             );
-            for (int i : new int[] { 10, 20, 1000 }) {
+            for (int i : new int[] { 10, 20, 990 }) {
+                // the test may stop after 1000 attempts, so might not report the 1000th failure; it definitely reports the 990th tho.
```

Comment:
Is the 1000 attempts some other variable/configuration?

Reply:
I see thanks! In that case shouldn't we verify 999 instead of 990?

Reply:
No, because we only report every 10th attempt:

https://github.com/elastic/elasticsearch/blob/95a215e5e0d5127ae102f247c55718f7f20cc6ee/server/src/main/java/org/elasticsearch/cluster/coordination/ElectionSchedulerFactory.java#L194

Reply:
Sounds good!